frontend/src/components/pages/profile/components/posts/Posts.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import EditIcon from '@material-ui/icons/Edit';
+import CloseIcon from '@material-ui/icons/Close';
+import UserPhoto from '../../../../mocks/user.png';
+import { Link } from 'react-router-dom';
+import './posts.scss';
+import { PostsList } from '../../../../interfaces/Interface';
+
+const Posts: React.FC<PostsList> = ({ posts }) => {
+  const editPost = (postId: number) => {
+    console.log(postId);
```

Comment:
instead of "console log" - create some file, which will be responsible for sending requests to backend, it will take some arguments and perform some requests via fetch.